### PR TITLE
refactor: dbot image imports

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by konstantinosG-deriv — APPROVED** [review]

**Review by eduard-deriv — APPROVED** [review]

**Comment by eduard-deriv** [comment]
LGTM!

File: src/images/common/dbot/index.ts
Comment (mayuran-deriv) [inline src/images/common/dbot/index.ts:1]: do we need this file?

**Review by mayuran-deriv — APPROVED** [review]

**Review by fasihali-deriv — DISMISSED** [review]

File: src/images/common/dbot/index.ts
Comment (Nuzhy-Deriv) [reply src/images/common/dbot/index.ts:1]: oh yeah. I missed to remove it

File: src/images/svg/dbot/index.ts
Comment (hasan-deriv) [inline src/images/svg/dbot/index.ts:1]: `export {default as DbotIcon} from './dbot-icon.svg'` Can you try this? Import and export in one link

**Review by prince-deriv — APPROVED** [review]

**Review by mitra-deriv — APPROVED** [review]

**Pull request merged**